oldschoolkeys/init.lua

```diff
@@ -4,24 +4,16 @@
 
 local pairs = pairs
 local lousy = require("lousy")
-local key, buf, but = lousy.bind.key, lousy.bind.buf, lousy.bind.but
-local add_binds, add_cmds = add_binds, add_cmds
 
-module("oldschoolkeys")
+local modes = require "modes"
 
-local userbindings = {
-    -- Normal mode:
-    normal = {
-        key({},          "b",           function (w, m) w:back(m.count)    end, {count=1}),
-        key({"Mod1"},    "Page_Up",     function (w, m) w.tabs:reorder(w.view, w.tabs:current() - m.count) end, {count=1}),
-        key({"Mod1"},    "Page_Down",   function (w, m) w.tabs:reorder(w.view, (w.tabs:current() + m.count) % w.tabs:count()) end, {count=1}),
-    },
-}
+local add_binds = modes.add_binds
+
+module("plugins.oldschoolkeys")
```

Comment:
The use of `module()` in plugins is now forbidden. This is because it unavoidably sets a global variable. Instead, declare a local table named `_M` and return it at the end of the file. All subsequent top-level variables should either be `local` or should be fields of `_M`.